packages/models/src/lib/only-plugins-options.ts

```diff
@@ -0,0 +1,11 @@
+import { z } from 'zod';
+
+export const onlyPluginsOptionsSchema = z.object({
```

Comment:
I wonder if this should live in `core`... 🤔 or in `cli` it is definitely more a cli than a models thing.
I don't really know where to put it. Spent already some time moving the CLI command options around... Maybe @matejchalk has good input?

Reply:
I don't believe this schema is being used anywhere, since it's not part of the core config. In which case, we only need the type definition, since only Yargs validates it. The type definition should belong in `core`, because that's where it's used. (If we wanted to allow `onlyPlugins` in the config file, then that would make it part of `models`.)